roles/lib_openshift/src/lib/base.py

```diff
@@ -264,7 +264,7 @@ def openshift_cmd(self, cmd, oadm=False, output=False, output_type='json', input
 
         rval = {}
         results = ''
-        err = None
+        error = None
```

Comment:
Yes, if the exception code path is hit, the exception uses err as the variable name for the exception and tries to set err in the method scope.